org-code-javabuilder/lib/src/main/java/org/code/javabuilder/OutputHandler.java

```diff
@@ -0,0 +1,30 @@
+package org.code.javabuilder;
+
+import java.nio.ByteBuffer;
+import com.amazonaws.services.apigatewaymanagementapi.AmazonApiGatewayManagementApi;
+import com.amazonaws.services.apigatewaymanagementapi.model.PostToConnectionRequest;
+
+public class OutputHandler extends Thread {
```

Comment:
Should this be a singleton? That seems somewhat discouraged yet common in the Java community.